metric-anomaly-data-model/src/main/proto/org/hypertrace/alert/engine/eventcondition/config/service/v1/metric_anomaly_event_condition.proto

```diff
@@ -77,6 +78,7 @@ enum Severity {
 message ViolationCondition {
   oneof condition {
     StaticThresholdCondition static_threshold_condition = 2;
+    DynamicThresholdCondition dynamic_threshold_condition = 3;
```

Comment:
I know, prior, we have used the same name - `DynamicThresholdCondition`. Do you think, `BaselineThresholdCondition` be a better name? Trying to be specific and bringing the word `baseline` in defintion. What do you think?